snmp/datadog_checks/snmp/data/default_profiles/_cisco-generic.yaml

```
@@ -192,6 +229,41 @@ metrics:
         table: entPhysicalTable
         tag: entity_name
 
+  - MIB: CISCO-STACKWISE-MIB
+    table:
+      OID: 1.3.6.1.4.1.9.9.500.1.2.1
+      name: cswSwitchInfoTable
+    symbols:
+      - name: cswSwitchInfo
+        constant_value_one: true
+    metric_tags:
+      - tag: mac_addr
+        column:
+          OID: 1.3.6.1.4.1.9.9.500.1.2.1.1.7
+          name: cswSwitchMacAddress
```

wondering if we should add `format: mac_address` here ?
```suggestion
      - tag: mac_addr
        column:
          OID: 1.3.6.1.4.1.9.9.500.1.2.1.1.7
          name: cswSwitchMacAddress
          format: mac_address
```

---

Good point since without this format we have `mac_addr:0x111111111111` but with the format it's `mac_addr:11:11:11:11:11:11`
It should be better having the second option

---

Got weird problem with tests, will revert some of the changes